pure2.md

```diff
@@ -0,0 +1,535 @@
+* PURE: 2
+* Title: Offline Updates
+* Version: 1
+* Last-Modified: 2021-09-16
+* Authors: Jon Oster <jon.oster@toradex.com>, Phil Wise <phil@phil-wise.com>
+* Status: Draft
+* Content-Type: <text/markdown>
+* Created: 2021-09-16
+
+# PURE-2: Offline Updates (Draft)
+
+## Abstract
+
+Firmware often needs to be updated offline, in situations where the full Uptane interactive flow is not possible for some reason--usually because internet connectivity is not available, or the device to be updated has not yet registered ECU keys with the Uptane backend. In those scenarios, it is still desirable to have some degree of validation and protection of the update process, and in particular it is desirable to ensure that these offline updates can be directed.
+
+This PURE offers a method of creating a bundle of firmware images and metadata that can be put on durable storage and used to install updates in the absence of connectivity with the update server, but without turning that bundle into a permanent vector for rollback attacks. To accomplish this, we define a new role on the Director repository for signing targets metadata for offline installation, along with the procedures for verifying this metadata.
+
+
+## Motivation
+
+NOTE:   Although Uptane was initially designed for automotive applications, it has been adopted for use in other IoT domains. Therefore, instead of referring to "vehicles" as the target of updates, this PURE refers more generically to "devices".
```

Comment:
I kind of wish this language was more generic everywhere in Uptane, although I imagine that the mental translation of "vehicles" to "devices" isn't so difficult.